create-neon/data/templates/README.md.hbs

```diff
@@ -1,5 +1,82 @@
 # {{package.name}}
 
 {{#if package.description}}
-{{package.description}}
+*{{package.name}}:* {{package.description}}
+
 {{/if}}
+This project was bootstrapped by [create-neon](https://www.npmjs.com/package/create-neon).
+
+## Building {{package.name}}
+
+Building {{package.name}} requires a [supported version of Node and Rust](https://github.com/neon-bindings/neon#platform-support).
+
+You can build the project with npm. In the project directory, run:
+
+```sh
+$ npm install
+```
+
+## Exploring {{package.name}}
+
+After building {{package.name}}, you can explore its exports at the Node REPL:
+
+```sh
+$ npm install
+$ node
+> require('.').hello()
+"hello node"
+```
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm install`
+
+Builds the project from source. 
+
+### `npm test`
+
+Runs the unit tests by calling `cargo test`. You can learn more about [adding tests to your Rust code](https://doc.rust-lang.org/book/ch11-01-writing-tests.html) from the [Rust book](https://doc.rust-lang.org/book/).
+
```

Comment:
Can you add `npm run build`? In this section, can you also explain what `cargo-cp-artifact` does?